playwright-junit/src/test/java/com/microsoft/playwright/junit/BrowserDefaultFactoryTests.java

```diff
@@ -0,0 +1,56 @@
+package com.microsoft.playwright.junit;
+
+import com.microsoft.playwright.Browser;
+import org.junit.jupiter.api.*;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@UseBrowserFactory
+public class BrowserDefaultFactoryTests {
+  private Browser browserFromBeforeEach;
+  private static Browser browserFromBeforeAll;
+
+  @BeforeAll
+  static void beforeAll(Browser browser) {
+    assert browser != null;
+    System.out.println("BeforeAll " + browser);
+    browserFromBeforeAll = browser;
+  }
+
+  @AfterAll
+  static void afterAll(Browser browser) {
+    assert browser != null;
+    System.out.println("AfterAll " + browser);
+    assertEquals(browserFromBeforeAll, browser, "Static Browser is not equal to others that were created");
```

Comment:
Is it guaranteed that @afterAll of BrowserExtension is called after all afterAll's in the test class?